src/app/patient-dashboard/visit/visit.component.ts

```diff
@@ -201,6 +201,9 @@ export class VisitComponent implements OnInit, OnDestroy {
             });
         }
     }
+    public retry() {
+        window.location.reload();
```

Comment:
This is expensive as it takes quite some time to rebuild the app. Would want to see how it behaves when there's a build